package-lock.json

```diff
@@ -88,7 +88,8 @@
         "react-dom": "18.2.0",
         "react-hook-form": "7.43.5",
         "react-redux": "8.1.2",
-        "react-router-dom": "6.8.2"
+        "react-router-dom": "6.8.2",
+        "stylelint": "15.10.2"
```

Comment:
Are you just changed version in package.json manually? Packages should be changed only via npm